packages/legend-manual-tests/src/__tests__/roundtrip-grammar/cases/DSLPersistence-basic.pure

```diff
@@ -9,14 +9,6 @@ Persistence org::legend::ZooPersistence
     sink: ObjectStorage
     {
       binding: org::legend::ZooBinding;
-      connection:
```

Comment:
This test doesn't seem to cover `PersistenceContext`. Should we add one for that use case? I'd prefer not to complicate this test file further, so let's add another file for that if appropriate. Thanks!

Reply:
Good catch.

Unfortunately, I think the <code>PersistenceContext</code> will require a valid <code>Persistence</code> and friends to compile. But will try to split up and see.